Code/Framework/AzCore/AzCore/RTTI/TypeInfo.h

```diff
@@ -508,6 +509,10 @@ namespace AZStd
     // GetO3deTypeName/GetO3deTypeId overload for AZStd::function
     AZ_TYPE_INFO_INTERNAL_SPECIALIZED_TEMPLATE_PREFIX_UUID(AZStd::function, "AZStd::function", "{C9F9C644-CCC3-4F77-A792-F5B5DBCA746E}", AZ_TYPE_INFO_INTERNAL_TYPENAME);
 
+    // GetO3deTypeName/GetO3deTypeId overload for AZStd::span<T, Extent>
+    // Note the Extent is not taking into account for template paramater is not taking into account and only the type T argument
```

Comment:
I'm not quite sure what this comment is trying to say, here's my best guess?
```suggestion
    // Note only the type T argument is taken into account as a template parameter, not the Extent
```

Reply:
Yeah that reads better.
The intent is that the `AZStd::span` type ID doesn't take into account the `Extent` template parameter.
So for example an `AZStd::span<AZ::Component*, dynamic_extent>` has the same type id as `AZStd::span<AZ::Component*, 100>`

Reply:
Oh, ok, I'd suggest the following then:
// Note the type ID only takes the type T argument into account, not the Extent.
// An AZStd::span<AZ::Component*, 50> and AZStd::span<AZ::Component*, 100> will have the same type ID.